aggregator_api/src/routes.rs

```diff
@@ -48,6 +48,8 @@ pub(super) async fn get_config(
             SupportedQueryType::TimeInterval,
             SupportedQueryType::FixedSize,
         ],
+        // Unconditionally indicate to divviup-api that we support collector auth token hashes
+        features: &["token-hash"],
```

Comment:
I have mild apprehension about introducing a feature flag for this, as if we have permanent/long-lived feature flags we risk combinatorial explosion of features down the line ([among other problems](https://bpapillon.com/post/feature-flags-theory-vs-reality/)).

I get the purpose is to avoid having to backport this feature (and we should avoid it IMO), but we should revisit this when we drop DAP-04. Perhaps can defer it with an issue for "remove `token-hash` feature flag?".

Reply:
divviup-api needs to dynamically figure out whether a Janus aggregator API it is working with needs collector auth token hashes or tokens. @jbr suggested not to infer this from `"protocol": "DAP-07"` since we might deploy this feature on other Janus versions, which I agree with. But you're right that we can eventually drop the feature: #2066